docs/get_started/getting_started_macos_bazel.md

```diff
@@ -89,11 +89,11 @@ by git. The recommended contents for Linux/macOS are:
 ```shell
 build --disk_cache=/tmp/bazel-cache
 
-# Use --config=debug to compile iree and llvm without optimizations
+# Use --config=debug to compile IREE and LLVM without optimizations
 # and with assertions enabled.
 build:debug --config=asserts --compilation_mode=opt '--per_file_copt=iree|llvm@-O0' --strip=never
 
-# Use --config=asserts to enable assertions in iree and llvm.
+# Use --config=asserts to enable assertions in IREE and LLVM.
 build:asserts --compilation_mode=opt '--per_file_copt=iree|llvm@-UNDEBUG'
```

Comment:
Pretty sure @bjacob said this would create ODR violations and was bad

Reply:
We can follow up on that later, separate from this PR.

Reply:
Yeah just noticed it :-)